apps/mymedicare_cb/authorization.py

```diff
@@ -59,127 +82,224 @@ def basic_auth(self):
             return (self.client_id, self.client_secret)
         return None
 
-    def exchange_for_access_token(self, req_token, request):
-        data_dict = {
-            "request_token": req_token,
-            "client_id": self.client_id,
-            "client_secret": self.client_secret,
-        }
+    def slsx_common_headers(self, request):
+        """
+        Common headers used for all endpoint requests
+        """
+        # keep using deprecated conv - no conflict issue
         headers = {"Content-Type": "application/json",
                    "Cookie": self.token_endpoint_aca_token,
                    "X-SLS-starttime": str(datetime.datetime.utcnow())}
 
         if request is not None:
             headers.update({"X-Request-ID": str(getattr(request, '_logging_uuid', None)
                             if hasattr(request, '_logging_uuid') else '')})
+        return headers
 
-        # Get auth flow trace session values dict.
-        auth_flow_dict = get_session_auth_flow_trace(request)
+    def exchange_for_access_token(self, req_token, request):
+        """
+        Exchnages the request_token from the slsx -> medicare.gov login
+        login flow for an auth_token via the slsx token endpoint.
```

Comment:
```suggestion
        flow for an auth_token via the slsx token endpoint.
```
Duplicate word on end of previous line